libcxx/include/__ranges/stride_view.h

```diff
@@ -391,8 +391,8 @@ namespace __stride {
 struct __fn {
   template <viewable_range _Range>
   [[nodiscard]] _LIBCPP_HIDE_FROM_ABI constexpr auto operator()(_Range&& __range, range_difference_t<_Range> __n) const
-      noexcept(noexcept(/**/ stride_view{std::forward<_Range>(__range), __n}))
```

Comment:
This isn't a fix arguably. We should instead put `//  clang-format off` around it. The point of the alignment is to show that all of the expressions are the exact same.

Please revert this.

Reply:
No problem! I was really worried -- I was trying to follow @var-const 's advice for formatting! I hope that everything looks okay!